.github/workflows/github-workflow.yml

```diff
@@ -0,0 +1,29 @@
+name: React Images Uploading
+# This workflow is triggered on pushes to the repository.
+on: [push]
+
+jobs:
+  test:
+    # Job name
+    name: Jest test
+    # This job runs on Linux
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        id: checkout
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install Packages
+        run: npm install
+      - name: Run Test
+        run: npm test
```

Comment:
should be `npm test -- --coverage`

Reply:
I think we should update test command on `package.json`

Reply:
if u runs unit test at local, it's not necessary to generate coverage by default and it takes time to generate at local which slows down development

Reply:
Normally, i'll separate the test commands to `test` and `test:covarage`. Doing so makes the command is clearer and shorter. How do you think about it ? :)

Reply:
jest itself already provided options for you --coverage. We don't have to add unnecessary scripts to package json, jest also has documentation for it. So we don't have to take care of the documentation. So I still think if we add script like u suggested is like we reinvent the wheel.

Reply:
Yah, I agree about Jest did provide the coverage option. From my point view, the command on `package.json` is just a project command (owned by us) and it has nothing to do with Jest command. We should make use of it for our own work

Reply:
Ok, here is the thing. I am not talking only in this project scope.

1. What if I add 1 more jest's option like `--changedSince` or `--onlyChanged`? Are you add one more command in `package.json`? Do you think it's a bit weird. And so on I add some more jest's options and you create each command for each options that you've added?

2. Each command you added, you need to add documentation while that documentation is already in jest documentation site. So every options you add to the code base, you have to update documentation. Is it useful? and then what if you decided you remove that option, are you gonna look for all of places to update documentation? Do you think it's a bit waste.

Reply:
Ok, got it! I have two points of view to share:

> 1. The final output that we expect are the test command to separated environment (dev/ prod ...). For that, only need separete the project command  (like things we're using:  `build` and `build:web`). When we need to add more flag, we still have to modify `package.json` or `github-workflow` file. I don't think modify `github-workflow` file is a good choice.

> 2. On the `github-workflow` file that's the `CI/CD` config file. The comment for every steps, every actions is make sense. Too much detail comment about the tools are included on it will make nosie.

Reply:
I didn't say we should modify the git action usually. What I am trying to say is make everything to be concise and take advantage the existing one instead adding many commands to `package.json`. We already have place to take a look (jest documentation), it's better to take advantage of it. What if jest change the description of specific options? with your solution => you have to modify our documentation. With my way, people just go jest documentation and see what changed and we don't have spend effort to maintain things that is already existed.

My point of view stands still. I prefer adding `npm test -- --coverage` instead of `npm run test:coverage` in `gitlab action config`.

Your idea & mine are in conflicts. Probably we need @vutoan266 to decide.